docs/configuration.md

```diff
@@ -9,8 +9,10 @@ When developing a Probot App, you will need to have several different fields in
 Variable | Description
 ---|---
 `APP_ID` | The App ID assigned to your GitHub App. **Required** <p>_(Example: `1234`)_</p>
-`PRIVATE_KEY_PATH` | The path to the `.pem` file for your GitHub App. If not present, Probot will look for a file ending in `.pem` in your project's directory. **Required** if the `.pem` file is not in your project directory and `PRIVATE_KEY` is not set. <p>_(Example: `path/to/key.pem`)_</p>
-`PRIVATE_KEY` | The contents of the private key for your GitHub App. **Required** if the `.pem` file is not in your project directory and `PRIVATE_KEY_PATH` is not set.
+**Private Key Options** | One of the following is **Required** if there is no `.pem` file in your project's root directory
+`PRIVATE_KEY_PATH` | The path to the `.pem` file for your GitHub App. <p>_(Example: `path/to/key.pem`)_</p>
+`PRIVATE_KEY` | The contents of the private key for your GitHub App. If you're unable to use multiline environment variables, use base64 encoding to convert the key to a single line string. | 
```

Comment:
> …use base64 encoding to convert the key to a single line string.

Any tips we can offer or docs we can link to on how to do that?

Reply:
I've been trying to think of the optimal way to present this. What I've been doing is opening up a node console and running `Buffer.from` on the key. I had a couple of ideas on how to make this better:

- Add a `script/encode-key` script in the template to uses `findPrivateKey()` to find the local key and adds it to `.env` automatically
- Add a commander command that does the same thing as ☝️ 
- Point people to one of the [online tools](https://www.base64encode.org/) that does it for them

Reply:
I see a `base64` command on my mac, but I'm not sure where it came from:

```
$ echo hello world | base64
aGVsbG8gd29ybGQK
```

Looks like it's from [coreutils](https://www.gnu.org/software/coreutils/coreutils.html), which I imagine is available on any mac or linux OS. Not sure about windows.

Reply:
I'm over here face palming....

`cat ./private-key.pem | base64` is what we recommended to set the variable in Now to begin with....

Windows is a different beast. There is [certutil](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/certutil), but it doesn't do a straight encoding to a single line string (which is the whole point). I'm inclined to just put a note that there's not a straight equivalent on Windows and if you can't use a Linux command line (Docker, WSL, git bash), use one of the online tools.